Autopilot/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ if (${KIND_OF_BUILD} STREQUAL "FOR_TARGET")
           -iex "disconnect" -iex "quit")
 
 elseif(${KIND_OF_BUILD} STREQUAL "UNIT_TESTS")
```

Comment:
In the future,  it's much better to merge the PR stuff into your branch (or rebase your branch onto the new PR). Copying changes manually like this might cause merge conflicts when the time comes.